Integrations/integration-ReversingLabs_A1000.yml

```diff
@@ -0,0 +1,452 @@
+            md += 'SHA512: **' + sha512 + '**\n'
+        if file_type:
+            file_data['Type'] = file_type
+        if file_info:
+            file_data['Info'] = file_info
+        if file_size:
+            file_data['Size'] = file_size
+
+        ec[outputPaths['file']] = file_data
+
+        md += 'ID: **{}**\n'.format(demisto.get(res, 'summary.id'))
+        md += 'Malware status: **{}**\n'.format(status)
+        md += 'Local first seen: **{}**\n'.format(res.get('local_first_seen'))
+        md += 'Local last seen: **{}**\n'.format(res.get('local_last_seen'))
+        md += 'First seen: **{}**\n'.format(demisto.gets(res, 'ticloud.first_seen'))
+        md += 'Last seen: **{}**\n'.format(demisto.gets(res, 'ticloud.last_seen'))
+        md += 'Trust factor: **{}**\n'.format(res.get('trust_factor'))
+        if status == 'malicious':
+            md += 'Threat name: **{}**\n'.format(res.get('threat_name'))
+            md += 'Threat level: **{}**\n'.format(res.get('threat_level'))
+        md += 'Category: **{}**\n'.format(res.get('category'))
+        md += 'Classification origin: **{}**\n'.format(res.get('classification_origin'))
+        md += 'Classification reason: **{}**\n'.format(res.get('classification_reason'))
+        md += 'Aliases: **{}**\n'.format(','.join(res.get('aliases')))
+        md += 'Extracted file count: **{}**\n'.format(res.get('extracted_file_count'))
+        md += 'File type: **{}/{}**\n'.format(file_type, file_info)
+        md += 'File size: **{}**\n'.format(file_size)
+        md += 'Identification name: **{}**\n'.format(res.get('identification_name'))
+        md += 'Identification version: **{}**\n'.format(res.get('identification_version'))
+        indicators = demisto.get(res, 'summary.indicators')
+        if indicators:
+            md += tableToMarkdown('Indicators', indicators)
+
+        demisto.results({'Type': entryTypes['note'], 'ContentsFormat': formats['json'], 'Contents': r, 'EntryContext': ec, 'HumanReadable': md})
+
+
+    def extracted_files():
+        """
+        Get the list of extracted files for a given sample
+        """
+        parent = demisto.getArg('hash')
+        endpoint = '/api/samples/{}/extracted-files/'.format(parent)
+        ok, r = validate_http(requests.get(base_url + endpoint, headers=headers))
+        if not ok:
+            return_error(r)
+
+        results = r.get('results')
+        if not results:
+            return_error('Reply does not contain results')
+
+        ec = {'DBotScore': []}
+        file_list = []
+        file_context_list = []
+        for res in results:
+            sha1 = demisto.get(res, 'sample.sha1')
+            status = demisto.get(res, 'sample.threat_status')
+            score = {'unknown': 0, 'known': 1, 'suspicious': 2, 'malicious': 3}[status]
+            prop = status.title()
+            file_data = {'SHA1': sha1, 'Name': res.get('filename'), 'Info': demisto.get(res, 'sample.type_display'),
```

Comment:
instead of 
`{'SHA1': sha1, 'Name': res.get('filename'), 'Info': demisto.get(res, 'sample.type_display')....`



prefer 
```
{
     'SHA1': sha1,
     'Name': res.get('filename'),
     'Info': demisto.get(res, 'sample.type_display'),
....
}
```